transforms/universal/resize/spark/README.md

```diff
@@ -0,0 +1,64 @@
+# Resize  files
+
+Please see the set of
+[transform project conventions](../../README.md)
+for details on general project conventions, transform configuration,
+testing and IDE set up.
+
+## Summary
+
+This is a simple transformer that is resizing the input tables to a specified size. 
+* resizing based on in-memory size of the tables.
+* resized based on the number of rows in the tables. 
+
+## Building
+
+A [docker file](Dockerfile) that can be used for building docker image. You can use
+
+```shell
+make build 
+```
+
+## Configuration and command line Options
+
+The set of dictionary keys holding [BlockListTransform](src/blocklist_transform.py)
+configuration for values are as follows:
+
+* _max_rows_per_table_ - specifies max documents per table
```

Comment:
To better future-proof this file, shouldn't it defer to the python readme for configuration and CLI?